android/app/src/androidTest/java/org/fossasia/openevent/DateTest.java

```diff
@@ -0,0 +1,87 @@
+package org.fossasia.openevent;
+
+import org.fossasia.openevent.utils.ISO8601Date;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by arpitdec5 on 02-06-2017.
+ */
+
+public class DateTest {
+
+    @Test
+    public void checkDate() {
```

Comment:
Ideally, there should be only one assert per test/function

Please break these into several tests and specify what it is expected to do
You can group similar assertions in one test

Look at this for reference - https://github.com/fossasia/open-event-orga-app/blob/master/app/src/test/java/org/fossasia/openevent/app/utils/UtilsTest.java

You can take a look at other test classes as well, there are plenty for reference